README.md

```diff
@@ -106,30 +107,30 @@ with LocalStack:
 
 #### Provisioning Infrastructure
 
-After starting LocalStack, create a terraform state deployment bucket in the localstack environment.
-This guide, as well as the provided Terraform backend file for local development, assumes that
+After starting LocalStack, create a terraform state deployment bucket in the localstack environment.  This guide, as well as the provided Terraform backend file for local development, assumes that
 this bucket is named `local-terraform` and has a region of `us-west-2`.
 
-To create this bucket, you can run the following command:
-
-```cli
-$ awslocal s3 mb s3://local-terraform
+To create this bucket, navigate to the terraform folder, then run the following command:
+```bash
+awslocal s3 mb s3://local-terraform
 ```
+If using the provided [docker-compose](https://github.com/usdigitalresponse/grants-ingest/blob/cf8a6220ed4e7332978520ec1d63bb344b738972/docker-compose.yml#L21) then this runs when the localstack 
+container is started. 
 
 Next, initialize the Terraform project using the `local.s3.tfbackend` Terraform state backend
 configuration file provided by this repository. Note that you may need to modify the `endpoint`
 setting in this file if your LocalStack environment uses a different host/port other than the
 default of `localhost:4566`.
 
-```cli
-$ tflocal init -backend-config="local.s3.tfbackend" -reconfigure
+```bash
+tflocal init -backend-config="local.s3.tfbackend" -reconfigure
 ```
 
-Once this command completes successfully, you use `tflocal` to "provision" mock infrastructure
-in your LocalStack environment, using the `local.tfvars` file provided by this repository:
-
-```cli
-$ tflocal apply -var-file=local.tfvars
+Once this command completes successfully, use `tflocal` to "provision" mock infrastructure in your LocalStack environment.  This will create the Lambda functions, so it requires the functions to be 
+built first.  Use `task build` to build the functions.  Then use the `local.tfvars` file provided by this 
```

Comment:
Just to mention: a side-effect of running `terraform plan` is generating builds for all the Lambdas being deployed (`terraform plan` calls `task build-<LambdaName>` under the hood). 

However, it is generally _much_ faster to build Lambdas ahead of time by running `task build` in the way you've documented here, so my preference is to keep this change.

Reply:
PS- Feel free to ignore the below, but in case you're wondering why `task build` is faster: 

`task build` batch-compiles all the lambda binaries, which lets the Go compiler optimize activities around fetching and linking common dependencies, whereas `terraform plan` runs `task build-<LambdaName>` for each handler (i.e. the difference between `go build ./cmd/...` vs `go build ./cmd/LambdaOne/... && go build ./cmd/LambdaTwo/...` etc.).

Although `terraform plan` will always run the per-Lambda `task` commands, our Taskfile is configured to short-circuit builds when the underlying source code is unchanged. Therefore, running `task build` ahead of time ensures everything is up-to-date in the most efficient way possible, and then subsequent `terraform plan` only build on an individual basis if anything has changed (and at that point, is generally fairly speedy).